src/features/Search/components/CoverCard/CoverCard.tsx

```diff
@@ -1,120 +1,112 @@
-        display="flex"
-        gridGap={2}
-        flexDirection="column"
-        w="full"
-      >
-        <Link
-          as={RouterLink}
-          variant="no-underline"
-          to={`/vn/${id}`}
-        >
-          <SafeImage
-            src={image}
-            containerProps={{
-              borderRadius: 'sm',
-            }}
-            isNsfw={isImageNsfw}
-            style={{ aspectRatio: '5 / 7' }}
-            w="full"
-            objectFit="cover"
-          />
-        </Link>
-        <Link
-          as={RouterLink}
-          variant="no-underline"
-          to={`/vn/${id}`}
-        >
-          <Heading
-            as="h2"
-            _groupHover={{
-              /* TODO(V1.8+): use chakra semantic tokens */
-              color: 'var(--color-link)',
-            }}
-            fontSize="sm"
-            fontWeight="medium"
-            noOfLines={2}
-          >
-            {title}
-          </Heading>
-        </Link>
-      </Box>
-    </PopoverTrigger>
-    <PopoverContent
-      p={4}
+  image,
+  title,
+  isImageNsfw,
+  link,
+  imagePlaceholder,
+  children,
+}) => {
+  const cardBox = (
+    <Box
+      role="group"
       display="flex"
+      gridGap={2}
       flexDirection="column"
-      gap={8}
-      width="min-content"
-      pointerEvents="none"
+      w="full"
     >
-      <Box
-        display="flex"
-        gap={12}
-      >
-        <CardDetail title="Released" text={released ? released.getFullYear() : DETAIL_DATA_NULL} />
-        <CardDetail title="Rating" text={rating} />
-        <CardDetail title="Length" text={length ?? DETAIL_DATA_NULL} />
-      </Box>
-      <Box
-        display="flex"
-        gap={12}
+      <Link
+        as={RouterLink}
+        variant="no-underline"
+        to={link}
       >
-        <CardDetailList
-          title="Languages"
-          items={languages}
-          readableMapper={Language.toReadable}
-          iconMapper={Language.getIcon}
+        <SafeImage
+          src={image}
+          fallbackSrc={imagePlaceholder}
+          containerProps={{
+            borderRadius: 'sm',
+          }}
+          isNsfw={isImageNsfw}
+          style={{ aspectRatio: '5 / 7' }}
+          w="full"
+          objectFit="cover"
         />
-        <CardDetailList
-          title="Platforms"
-          items={platforms}
-          readableMapper={Platform.toReadable}
-          iconMapper={Platform.getIcon}
-        />
-      </Box>
-    </PopoverContent>
-  </Popover>
-);
+      </Link>
+      <Link
+        as={RouterLink}
+        variant="no-underline"
+        to={link}
+      >
+        <Heading
+          as="h2"
+          _groupHover={{
+          /* TODO(V1.8+): use chakra semantic tokens */
+            color: 'var(--color-link)',
+          }}
+          fontSize="sm"
+          fontWeight="medium"
+          noOfLines={2}
+        >
+          {title}
+        </Heading>
+      </Link>
+    </Box>
+
+  );
+
+  if (children != null) {
```

Comment:
Maybe, just make `children` a required prop?

Reply:
You missed the point, there I don't render popover if children not provided.

Reply:
Add proper docs for `children` then 🙂